internal/bot/bot.go

```diff
@@ -226,7 +229,7 @@ func (b *Bot) createAnnotations(ctx context.Context, pr *github.PullRequest, r *
 
 	for _, f := range parsedDiff.Files {
 		// Skip over any files listed in `ignore`
-		if ignoredByRepo(ctx, r, headSHA, f.NewName, ghc) {
+		if ignoredByRepo(rc, f.NewName) {
```

Comment:
i'd much prefer `repoConfig` and `repo` above. But you can address that another time. looking at the remove side of this line i was very confused. Also, `ghc` is also the Glasgow Haskell Compiler 😄 so i got rlly confused